{{cookiecutter.project_slug}}/.dockerignore

```diff
@@ -1,13 +1,75 @@
 postgres-data/
 .env
 *.pyc
-*.mo
 *.db
 *.css.map
-.cache
 .sass-cache
-.idea
 .vagrant
 .git
 .DS_Store
-__pycache__
\ No newline at end of file
+
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
+
+# Distribution / packaging
+.Python
+env/
+build/
+develop-eggs/
+dist/
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
+parts/
+sdist/
+var/
+*.egg-info/
+.installed.cfg
+*.egg
+
+# PyInstaller
+#  Usually these files are written by a python script from a template
+#  before PyInstaller builds the exe, so as to inject date/other infos into it.
+*.manifest
+*.spec
+
+# Installer logs
+pip-log.txt
+pip-delete-this-directory.txt
+
+# Unit test / coverage reports
+htmlcov/
+.tox/
+.coverage
+.coverage.*
+nosetests.xml
+coverage.xml
+*,cover
+
+# Translations
+*.pot
+
+# Django stuff:
+*.log
+
+# Sphinx documentation
+docs/_build/
+
+# PyBuilder
+target/
+
+# Docker
+Dockerfile
+docker-compose.yml
+docker-compose.override.yml
+docker/docker-compose.prod.yml
```

Comment:
Why is this excluding docker related files?

Reply:
@driverdan The image built doesn't necessarily need to have the docker-related files since we won't actually be running `docker *` commands from the containers created from it.

Reply:
Oh of course, I misread this and thought it was .gitignore, not .dockerignore 🤦